core/storage/exploration/gae_models_test.py

```diff
@@ -432,32 +461,6 @@ def test_revert_to_version_with_invalid_status_is_successful(self):
             new_collection_model.to_dict(exclude=self.excluded_fields)
         )
 
-    def test_revert_to_version_with_translator_ids_field_is_successful(self):
-        broken_dict = dict(**self.original_dict)
-        del broken_dict['voice_artist_ids']
-        broken_dict['translator_ids'] = [self.USER_ID_2]
-
-        snapshot_model = (
-            exp_models.ExplorationRightsSnapshotContentModel
-            .get_by_id(
-                exp_models.ExplorationRightsModel.get_snapshot_id(
-                    self.EXPLORATION_ID_1, 1))
-        )
-        snapshot_model.content = broken_dict
-        snapshot_model.update_timestamps()
-        snapshot_model.put()
-        with self.allow_revert_swap, self.allowed_commands_swap:
-            exp_models.ExplorationRightsModel.revert(
```

Comment:
This actually still seems like a worthwhile flow since it's actually ensuring ``convert_to_valid_dict`` is being called when expected. Could you also add a test to verify that reversion properly excludes the two deprecated fields even when reverting to an older version that includes those fields?

@vojtechjelinek as FYI since I'm guessing this will be replicated in other similar migration PRs.

Reply:
@BenHenning I didn't get your point. Can you please explain in detail? Thanks!

Reply:
```
def test_revert_to_check_deprecated_fields_are_absent(self):
         with self.allow_revert_swap, self.allowed_commands_swap:
            exp_models.ExplorationRightsModel.revert(
                self.exploration_model, self.USER_ID_COMMITTER, 'Revert', 1)

            exp_rights_model = (
                exp_models.ExplorationRightsModel.get_by_id(
                    self.EXPLORATION_ID_1))

            snapshot_dict = exp_rights_model.compute_snapshot()

            self.assertNotIn('translator_ids', snapshot_dict)
            self.assertNotIn('all_viewer_ids', snapshot_dict)
```
Are you talking about this? It validates that these fields are not present in older versions

Reply:
Yes, the new test you added covers the behaviors well. Thanks!